tests/Microsoft.Identity.Test.Unit/ExceptionTests/MsalExceptionTests.cs

```diff
@@ -67,6 +68,49 @@ public void MsalClientException_FromMessageAndCode()
             Assert.IsFalse(piiMessage.Contains(ExMessage));
         }
 
+        [TestMethod]        
+        public void IsRetryable()

```

Comment:
Sadly we didn't write unit tests when we added this property in the first place....